test/pylib/manager_client.py

```diff
@@ -191,6 +192,14 @@ async def server_add(self, replace_cfg: Optional[ReplaceConfig] = None,
                 data['property_file'] = property_file
             if expected_error:
                 data['expected_error'] = expected_error
+
+            # If we replace a node and do not expect errors, we wait until other nodes
+            # see the node being replaced as dead. The replace operation may be
+            # rejected if the node being replaced is considered alive.
+            if replace_cfg and not expected_error:
```

Comment:
I don't like that we are making the wait conditional on whether we expect an error or not. It's not obvious that it's always what we want - what if we want to write a different test than `test_replacing_alive_node_fails` later which expects replace to fail, but not due to the node still being seen as alive by others?

Instead, I think we should extend `ReplaceConfig` to allow choosing whether we want to wait or not; alternatively, add a parameter to `add_server` which allows to control it.

Reply:
Sure, I'll change it. I think I'll take the second approach. I don't think this waiting is logically a part of the replace config. Also, `server_start` and `server_restart` already have a similar `wait_others` parameter.

Reply:
Done.